fastly/validators.go

```diff
@@ -27,6 +27,13 @@ func validateLoggingPlacement() schema.SchemaValidateFunc {
 	}, false)
 }
 
+func validateLoggingServerSideEncryption() schema.SchemaValidateFunc {
+	return validation.StringInSlice([]string{
+		"AES256",
+		"aws:kms",
```

Comment:
It would be good to use the `gofastly` constants here such as `gofastly.S3ServerSideEncryptionAES`, like you have in the switch statement above.